pdesolvers/heat_equation_1d/main.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from heat_equation import HeatEquation
+from pdesolvers.heat_equation_1d.solver.cn_solver import HeatEquationCNSolver
+from pdesolvers.heat_equation_1d.solver.explicit_solver import HeatEquationExplicitSolver
+
+
+def main():
+
+    equation = (((HeatEquation(1, 100,30,10000, 0.01)
```

Comment:
I'm a little confused with the parentheses here.

Reply:
it just allows me to format the .function_call